src/assets/create.js

```diff
@@ -19,11 +19,9 @@ class UI {
          </div>
       `;
 
-      document.querySelectorAll('.input').forEach((checked, i) => {
-        const taskName = checked.nextElementSibling;
-        checked.addEventListener('change', () => {
+      document.querySelectorAll('.input').forEach((complete, i) => {
+        complete.addEventListener('change', () => {
           const listItems = Storage.getList();
```

Comment:
- [ ] The user should get the correct list values after the page reloads. Please ensure the tasks that have the completed value set to true are checked after the page reloads.